atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/async/CassandraAsyncKeyValueServiceTests.java

```diff
@@ -109,6 +114,24 @@ public void testAllVisible() throws Exception {
         assertThat(result).containsOnlyKeys(VISIBLE_CELL_1, VISIBLE_CELL_2);
     }
 
+    @Test
+    public void testIsValidFalseWhenClientContainerClose() {
+        when(cqlClientContainer.isClosed()).thenReturn(true);
+        assertThat(asyncKeyValueService.isValid()).isFalse();
+    }
+
+    @Test
+    public void testIsValidFalseWhenClientIsInvalid() {
```

Comment:
(Believe I actually want `isClosed` to be false for this test, since otherwise the `isValid` check returns false since the container is closed, without even checking the cql client).

Regardless, have made the mock explicit.